README.md

```diff
@@ -4,47 +4,41 @@
 [![codecov](https://codecov.io/gh/dojo/widget-core/branch/master/graph/badge.svg)](https://codecov.io/gh/dojo/widget-core)
 [![npm version](https://badge.fury.io/js/%40dojo%2Fwidget-core.svg)](https://badge.fury.io/js/%40dojo%2Fwidget-core)
 
-This repo provides users with the ability to write their own Dojo 2 widgets.
-
-We also provide a suite of pre-built widgets to use in your applications: [(@dojo/widgets)](https://github.com/dojo/widgets).
-
-**WARNING** This is _beta_ software. While we do not anticipate significant changes to the API at this stage, we may feel the need to do so. This is not yet production ready, so you should use at your own risk.
-
-- [Usage](#usage)
-- [Features](#features)
-    - [Overview](#overview)
-    - [`v` & `w`](#v--w)
-        - [`v`](#v)
-        - [`w`](#w)
-    - [tsx](#tsx)
-    - [Writing custom widgets](#writing-custom-widgets)
-        - [Public API](#public-api)
-        - [The 'properties' lifecycle](#the-properties-lifecycle)
-            - [Individual Property Diffing](#individual-property-diffing)
-			- [Property Diffing Reactions](#property-diffing-reactions)
-		- [Render Lifecycle](#render-lifecycle)
-        - [Projector](#projector)
-		  - [Server Side Rendering](#server-side-rendering)
-        - [Event Handling](#event-handling)
-        - [Widget Registry](#widget-registry)
-        - [Injecting State](#injecting-state)
-        - [Theming](#theming)
-        - [Internationalization](#internationalization-i18n)
-        - [Web Components](#web-components)
-            - [Attributes](#attributes)
-            - [Properties](#properties)
-            - [Events](#events)
-            - [Initialization](#initialization)
-        - [Meta](#meta)
-		- [DomWrapper](#domwrapper)
-    - [Key Principles](#key-principles)
-    - [API](#api)
+widget-core is library to create powerful, composable user interface widgets.
+
+* **Reactive & unidirectional:** Widget-core follows core reactive principles to ensure predictable consistent behavior.
+* **Encapsulated Widgets:** Create independent encapsulated widgets that can be wired together to create complex and beautiful user interfaces.
+* **DOM Abstractions:** Widget-core provides API abstractions to avoid needing to access or manipulate the DOM outside of the reactive render life-cycle.
+* **I18n & Themes:** Widget-core provides core mixins to enable internationalization and theming support for your components.
+
+-----
+
+ - [Installation](#installation)
+ - [Features](#features)
+ 	- [Basic Widgets](#basic-widgets)
+		- [Rendering a Widget in the DOM](#rendering-a-widget-in-the-dom)
+		- [Widgets and Properties](#widgets-and-properties)
+		- [Composing Widgets](#composing-widgets)
+		- [Extracting Widgets](#extracting-widgets)
```

Comment:
Decomposing or destructuring instead of extracting?  Extracting seems like you are going to be taking references out of a larger widget.